src/Notes/WC_Admin_Notes_Draw_Attention.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * WooCommerce Admin Draw Attention note provider
+ *
+ * Adds a note to the merchant's inbox
+ *
+ * @package WooCommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Notes_Draw_Attention
+ */
+class WC_Admin_Notes_Draw_Attention {
+	const NOTE_NAME   = 'wc-admin-draw-attention';
+	const OPTION_NAME = 'woocommerce_onboarding_profile';
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		// Trigger this when the onboarding options are updated.
+		add_filter(
+			'update_option_' . self::OPTION_NAME,
```

Comment:
Love this way of watching for the change ❤️

Reply:
Thanks, it came to me during a nap ;-)